src/sharing/types.ts

```diff
@@ -11,12 +11,13 @@ export interface SharingRule {
 
 export interface SharedObject {
     id: Id;
+    name: string;
```

Comment:
`name` as a required prop? since the expression using it is `displayName || name`, I guess you wanted it to make optional, right?

Reply:
Actually it makes sense to me that ``displayName`` isthe optional one and if defined used and otherwise use the required ``name``